scss/_base_tables.scss

```diff
@@ -71,6 +71,6 @@
   // stylelint-enable selector-max-type
 
   %table-row-border {
-    border-top: 1px solid $colors--theme--border-low-contrast;
+    border-top: $input-border-thickness solid $colors--theme--border-default;
```

Comment:
@britneywwc Why changing this to `$input-border-thickness`? 

Overall the fix works fine, but I'm worried that changing the width of the border on ALL rows may affect other things.

Reply:
I tried to match the border thickness with the table-mobile-card's border which inherits from `$border`. 

Would it be better to change it in `p-table--mobile-card` rather than `%table-row-border` so that it uses the same border thickness (1px) as the previous table rows? Just so it won't affect other things and only change within the table-mobile-card component.

Reply:
Yes, I think we need to treat it separate from standard table row border.

In mobile card small screen view we want those to look like card component, so the border around each card (row) should be the same as in card, so as you mentioned it should be `$border`

As in:

```
  %vf-is-bordered {
    border: $border;
  }
```